### ISSUE TITLE
Android?

### ISSUE THREAD

**Issue by fragmental** [chadaustin/is-it-snappy#10]
Hello.
I am wondering if you are aware of an application like this made for Android?  Or perhaps have considered porting this to Android?  I don't have an iphone, but I do have an android phone with a 240fps camera.  

I would consider porting it myself, but I don't know anything about iOS programming, or objective C, or Swift.  Neither do I know about Android programming, Java, or Kotlin.  My programming experience is mostly with C#.  I have found two Swift to Kotlin converters [here](https://github.com/angelolloqui/SwiftKotlin) and [here](https://vinivendra.github.io/Gryphon/).  Gryphon seems to make some very bold claims about being able to compile translated code, while SwiftKotlin's claims are more conservative.  It looks like both have recent commits.

**Comment by chadaustin** [comment]
I don't have an Android phone myself and certainly don't do Android development, so I think someone would just have to develop an equivalent app for Android.

It would be very hard to do a direct port of Is It Snappy? as it relies heavily on iOS APIs. Fortunately it's a simple app. :)

**Comment by fragmental** [comment]
Thank you for your quick response.  As much as I would love to  have an app like this for android, it's well beyond my technical skill to create one, atm.  It's tempting to think about, though.  Surely "a simple app" means it would be easy for me, right :D

Besides wanting to test my latency just to know what it is, I would ultimately like to use it to recreate "original hardware with crt" latency in Retroarch using Run-Ahead and Frame Delay.  Perhaps I should first try to record it just by checking it manually with a video?

I will let you decide whether to keep the issue open, or close it.

**Comment by chadaustin** [comment]
Yeah, you always have the option of recording slow-mo video files and using a PC to count the frames between events!

I'd be curious to hear how low you can get the latency in your project. :)
